library/linux/src/internal/key_event_handler.cc

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include "library/linux/src/internal/key_event_handler.h"
+#include "library/linux/src/internal/json_message_codec.h"
```

Comment:
This header should be in its own section after the library headers, per Google style. Only the associated header (key_event_handler.h) goes above.